Wikipedia/Code/WMFFindInPageKeyboardBar.m

```diff
@@ -109,10 +109,13 @@ - (void)updateLabelTextForCurrentMatchIndex:(NSInteger)index matchesCount:(NSUIn
         labelText = nil;
     } else if (count > 0 && index == -1) {
         labelText = [NSString stringWithFormat:@"%lu", (unsigned long)count];
-    } else if (count == 0) {
-        labelText = [NSString stringWithFormat:MWLocalizedString(@"find-in-page-number-matches", nil), 0, 0];
     } else {
```

Comment:
I also removed this unnecessary `else if` block. The `else` clause catch it, `count` and `index + 1` are equal to 0.